quiz/web/src/router.tsx

```diff
@@ -1,4 +1,5 @@
 import { PlayerRequired } from "modules/common/components";
+import Start from "modules/Start/Start";
```

Comment:
Shorten it to `import Start from "modules/Start"`

Reply:
fixed